config/initializers/algoliasearch.rb

```diff
@@ -0,0 +1,9 @@
+AlgoliaSearch.configuration = {
+	application_id: "YE495E6TKG",
+	api_key: "21c4fe4689e99b41c2ca180c74cbec6a",
```

Comment:
You might consider rolling these keys now that they have been shared in a public repo.